packages/web-components/fast-foundation/src/picker/picker.ts

```diff
@@ -94,22 +94,58 @@ export class FASTPicker extends FormAssociatedPicker {
     }
 
     /**
-     * Whether the component should remove an option from the list when it is in the selection
+     * DEPRACATED -- use disableSelectionFilter. Whether the component should remove an option from the list when it is in the selection
```

Comment:
If you use `@deprecated` it should tag and pick this up in API Extractor

Reply:
done